crates/bevy_ecs/src/schedule_v3/migration.rs

```diff
@@ -0,0 +1,34 @@
+use crate::schedule_v3::*;
+use crate::world::World;
+
+/// New "stageless" `App` methods.
+pub trait AppExt {
+    /// Sets the [`Schedule`] that will be modified by default when you call `App::add_system`
+    /// and similar methods.
+    ///
+    /// **Note:** This will create the schedule if it does not already exist.
+    fn set_default_schedule(&mut self, label: impl ScheduleLabel) -> &mut Self;
+    /// Applies the function to the [`Schedule`] associated with `label`.
+    ///
+    /// **Note:** This will create the schedule if it does not already exist.
+    fn edit_schedule(
```

Comment:
I thought I had both covered. Does the `set_default_schedule` method not fit the first style?

I like the closure because you can't forget to change it back. Reduce the chances of you accidentally making your plugins dump their systems in the wrong schedule, then having to spend time figuring it out.

Reply:
tbh I don't have a strong preference here. My only worry is that closures can be confusing to beginners, But this probably isn't a beginner API. So probably not worth worrying about too much.

As a separate comment, are the sub app schedules going to end up living in `Schedules`?

Reply:
> As a separate comment, are the sub app schedules going to end up living in `Schedules`?

Not sure how to answer you, but the I'm pretty sure the logical hierarchy is like this:
```
App
- SubApp
  - World
    - Schedule
```

Reply:
Thanks, that answers my question.